packages/website/pages/docs/resources/taiko-glossary.md

```diff
@@ -0,0 +1,95 @@
+## Anchor Transaction
+
+The first transaction in every Taiko L2 block to perform data validation and enable L1-to-L2 communication.
+
+## Bridge
+
+The Taiko bridge allowing users to bridge Ether and custom ERC-20 tokens from and to Taiko.
+
+## Commit Hash
+
+The Keccak-256 hash of a proposed block’s beneficiary and transaction list.
+
+## Cross-chain communication (messaging)
+
+Bridging over messages using a pair of smart contracts that use state proofs to verify data across chains.
+
+## Decentralization
+
+The proposing and proving design where no single party is able to control all transaction ordering or is solely responsible for proving blocks. Being sufficiently decentralized implies that the protocol should keep working in a reliable manner in adversarial situations.
+
+## Deterministic
+
+The block execution is deterministic after the block is appended to the proposed block chain, all block properties are immutable and the post-block state can be calculated by anyone.
+
+## EtherVault
+
+A contract that holds some huge amount of ETHs on L2 and is able to authorize third party bridges to receive Ether from the EtherVault to execute deposit operations.
+
+## Fork Choice
+
+Because blocks can be proven in parallel, the Taiko smart contract can receive a proof for a block for which it doesn’t know the correct pre-state for. The smart contract thus accepts all valid proofs using different pre-states, once the smart contract know the correct pre-state as proofs for all parent blocks come in, the correct pre-state can be selected together with its corresponding post-state to advance the verified chain.
+
+## Golden Touch Address
+
+An address with a revealed private key to transact all [anchor transactions](#Anchor-Transaction).
+
+## Incentive multipliers
+
+Time sensitive multipliers that can decrease fees and increase rewards to incentivize proposals and proofs when there are unpredictable deterrents acting against the engagement of proposers or provers.
+
+## Intrinsic Validity Function
+
+A function that is run on each proposed block to determine if the block contains valid data. If the block doesn’t contain valid data the block will be seen as a block with an empty transaction list.
+
+## Invalid Block
```

Comment:
same here sentence case. i sort of believe we can remove this though, it is more of a "concept"